src/AWS.elm

```diff
@@ -80,35 +95,50 @@ type Request a
     = UnsignedRequest (AWS.Http.UnsignedRequest a)
 
 
-{-| Alias for Http.Request
+{-| Leaves the default options unchanged.
 
-See http://package.elm-lang.org/packages/elm-lang/http/1.0.0/Http#Request
+Alias for the identity function.
 -}
-type alias SignedRequest a =
-    Http.Request a
+defaultOptions : a -> a
+defaultOptions options =
+    options
 
 
-{-| Sign an unsigned request, given the current time
--}
 sign :
     ServiceConfig
     -> Credentials
     -> Date
     -> Request a
-    -> Result String (SignedRequest a)
+    -> Http.Request a
```

Comment:
I started down the road of implementing the work in this PR myself to test S3 calls (albeit, not as cleanly), and having this function return a `Result` was a bit cumbersome. This new signature definitely helps 👍